adafruit_tfmini.py

```diff
@@ -99,18 +101,18 @@ def strength(self) -> int:
         return self._strength
 
     @property
-    def mode(self) -> int:
+    def mode(self) -> Literal[2, 7]:
         """The measurement mode can be MODE_SHORT (2) or MODE_LONG (7)"""
         _ = self.distance  # trigger distance measurement
         return self._mode
 
     @mode.setter
-    def mode(self, newmode: int) -> None:
+    def mode(self, newmode: Literal[MODE_LONG, MODE_SHORT]) -> None:
```

Comment:
I don't believe that using variables is valid use of `Literal`  Let's make it `Literal[2, 7]` like the getter.